Dockerfile

```diff
@@ -18,7 +18,7 @@
 # These ARGs must be before the first FROM. This allows them to be valid for
 # use in FROM instructions.
 ARG NNFMFU_TAG_BASE=ghcr.io/nearnodeflash/nnf-mfu
-ARG NNFMFU_VERSION=v0.0.2
+ARG NNFMFU_VERSION=0.0.2
```

Comment:
This makes me think that I've done something incorrect with docker's `--build-arg` and the `ARG` keyword, because this 0.0.2 value should have been passed in from the Makefile.   But I'll leave that for another day.